superset/connectors/sqla/models.py

```diff
@@ -930,9 +938,8 @@ def _get_sqla_row_level_filters(
         filters_grouped: Dict[Union[int, str], List[str]] = defaultdict(list)
         try:
             for filter_ in security_manager.get_rls_filters(self):
-                clause = text(
-                    f"({template_processor.process_template(filter_.clause)})"
-                )
+                rendered_filter = template_processor.process_template(filter_.clause)
+                clause = text(f"({rendered_filter})")
```

Comment:
```suggestion
                clause = text(f"({template_processor.process_template(filter_.clause)})")
```

Maybe just inline this given that `rendered_filter` isn't used elsewhere.